packages/wonder-blocks-timing/src/components/with-action-scheduler.js

```diff
@@ -21,12 +21,16 @@ type WithoutActionScheduler<T> = $Exact<$Diff<T, WithActionSchedulerProps>>;
  * these props use the `WithActionScheduler` type.
  */
 export default function withActionScheduler<
-    Config: {...},
-    Component: React.ComponentType<WithActionScheduler<Config>>,
+    -Config: WithActionSchedulerProps,
+    +Instance,
+    Component: React.AbstractComponent<WithActionScheduler<Config>, Instance>,
 >(
     WrappedComponent: Component,
-): React.ComponentType<WithoutActionScheduler<React.ElementConfig<Component>>> {
-    return React.forwardRef((props, ref) => (
+): React.AbstractComponent<
+    WithoutActionScheduler<React.ElementConfig<Component>>,
+    Instance,
+> {
+    return React.forwardRef<_, Instance>((props, ref) => (
```

Comment:
These changes mean that when ref forwarding is used, instead of the ref being `mixed` it will be typed to the wrapped instance.